redcap_bridge/project_building.py

```diff
@@ -128,7 +133,6 @@ def customize_project(project_built_csv, customization_csv, output_file=None):
     project_df.index = project_df['Variable / Field Name']
     customization_df = pd.read_csv(customization_csv, dtype=str)
     customization_df.index = customization_df['Variable / Field Name']
-
```

Comment:
I think removing this empty line does not improve readability of the code... could you reverse this change?